contrib/completion/zsh/_docker

```diff
@@ -231,6 +231,17 @@ __docker_log_options() {
     return ret
 }
 
+__docker_complete_detach_keys() {
+    [[ $PREFIX = -* ]] && return 1
+    integer ret=1
+
+    compset -P "*,"
+    keys=($(echo {a..z}))
+    ctrl_keys=($(echo ctrl-{{a..z},{@,'[','\\','^',']',_}}))
```

Comment:
echo is frowned upon in zsh community, better use print or printf. In your case, just use:

```
ctrl_keys=(${:-ctrl-{{a..z},{@,'[','\\','^',']',_}}})
```